h/static/scripts/threading-service.coffee

```diff
@@ -0,0 +1,97 @@
+class ThreadingService
+  # Mix in message thread properties into the prototype. The body of the
+  # class will overwrite any methods applied here. If you need inheritance
+  # assign the message thread to a local varible.
+  $.extend(this.prototype, mail.messageThread())
```

Comment:
It's not immediately obvious to me where `mail` came from. Could you add a comment pointing the reader to `jwz.js`? I know this will be clearer once we browserify things, but for the moment it's a bit magic.